src/components/RenderFakeData.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import moment from 'moment'
+// import * as FakeBookings from "../data/fakeBookings.json";
+
+const RenderFakeData = (props) => {
+
+return  props.results.map(function (jsonData , i){
+    return (
+      <tr key={i}>
+        <td>{jsonData.id}</td>
+        <td>{jsonData.title}</td>
+        <td>{jsonData.firstName}</td>
+        <td>{jsonData.surname}</td>
+        <td>{jsonData.email}</td>
+        <td>{jsonData.roomId}</td>
+        <td>{jsonData.checkInDate}</td>
+        <td>{jsonData.checkOutDate}</td>
+        <td>{moment(jsonData.checkOutDate).diff(jsonData.checkInDate, 'days')}</td>
+      </tr> 
+
+    )
+  });
+
+}
+
+  // const TableRow = function(props) {
```

Comment:
Should leave the commented out code from the final version, if you use Git it should be in the history if you ever need it back